python/packages/nisar/workflows/stage_watermask.py

```diff
@@ -455,7 +551,7 @@ def check_watermask_overlap(watermaskFilepath, polys):
     return perc_area
 
 
-def check_aws_connection():
+def check_aws_connection(version='0.5'):
```

Comment:
WDYT about removing the default value? That way, if we release new versions of the Water Mask Product, we won't need to remember to update this.

```suggestion
def check_aws_connection(version):
```